diff --git a/ducklib/thirdparty/backward.hpp b/ducklib/thirdparty/backward.hpp
index 36eafe3..e01a8bf 100644
--- a/ducklib/thirdparty/backward.hpp
+++ b/ducklib/thirdparty/backward.hpp
@@ -3973,9 +3973,8 @@ public:
   int inliner_context_size;
   int trace_context_size;
 
-  Printer()
-      : snippet(true), color_mode(ColorMode::automatic), address(false),
-        object(false), inliner_context_size(5), trace_context_size(7) {}
+  Printer(bool snippet = true, ColorMode::type color_mode = ColorMode::automatic, bool address = false, bool object = false, int inliner_context_size = 5, int trace_context_size = 7)
+  	: snippet(snippet), color_mode(color_mode), address(address), object(object), inliner_context_size(inliner_context_size), trace_context_size(trace_context_size) {}
 
   template <typename ST> FILE *print(ST &st, FILE *fp = stderr) {
     cfile_streambuf obuf(fp);
